### PR TITLE
Device types

### PR DESCRIPTION
**Pull request by CityVibes**
Added more device types as per #20 issue.

### CODE REVIEW

**Review by idokleinman — APPROVED** [review]

File: cloudsdk/src/main/java/io/particle/android/sdk/cloud/ParticleDevice.java [hunk 1]
```diff
@@ -43,14 +43,29 @@
...
                     return ELECTRON;
+                case 31:
+                    return RASPBERRY_PI;
+                case 82:
+                    return DIGISTUMP_OAK;
+                case 88:
+                    return RED_BEAR_DUO;
+                case 103:
+                    return BLUZ;
                 case 5:
```
Comment (idokleinman) [inline cloudsdk/src/main/java/io/particle/android/sdk/cloud/ParticleDevice.java]: 5 is not a Photon/P0 platform ID, it's only 6 AFAIK

@towynlin  right?

**Review by idokleinman — CHANGES_REQUESTED** [review]

File: cloudsdk/src/main/java/io/particle/android/sdk/cloud/ParticleDevice.java (hunk 1 above)
Comment (towynlin) [reply cloudsdk/src/main/java/io/particle/android/sdk/cloud/ParticleDevice.java]: 6 only

Comment (CityVibes) [reply cloudsdk/src/main/java/io/particle/android/sdk/cloud/ParticleDevice.java]: What is 5 then?

**Review by jensck — APPROVED** [review]
LGTM

File: cloudsdk/src/main/java/io/particle/android/sdk/cloud/ParticleDevice.java (hunk 1 above)
Comment (idokleinman) [reply cloudsdk/src/main/java/io/particle/android/sdk/cloud/ParticleDevice.java]: Nothing :)

Comment (CityVibes) [reply cloudsdk/src/main/java/io/particle/android/sdk/cloud/ParticleDevice.java]: @idokleinman Removed it.

**Comment by CityVibes** [comment]
@idokleinman Any changes to do, or can I merge it?

**Review by idokleinman — APPROVED** [review]

**Pull request merged**